multimodal/tarko/agent-web-ui/src/standalone/chat/Message/components/ToolCalls.tsx

```diff
@@ -150,11 +150,16 @@ export const ToolCalls: React.FC<ToolCallsProps> = ({
         case 'run_command':
           return args.command || (status === 'constructing' ? 'preparing command...' : '');
         case 'read_file':
+          if (args.path) {
+            const normalizedPath = normalizeFilePath(args.path);
+            return `reading: ${normalizedPath}`;
+          }
+          return status === 'constructing' ? 'preparing file operation...' : '';
         case 'write_file':
+        case 'edit_file':
           if (args.path) {
             const normalizedPath = normalizeFilePath(args.path);
-            const fileName = normalizedPath.split(/[/\\]/).pop();
-            return `file: ${fileName}`;
+            return `${toolCall.function.name === 'write_file' ? 'writing' : 'editing'}: ${normalizedPath}`;
```

Comment:
Remove this prefix: ${toolCall.function.name === 'write_file' ? 'writing' : 'editing'}